spec/unit/solr_doc_hash_spec.rb

```diff
@@ -1,420 +1,399 @@
-        :display_type => 'zzzz', 
-        :building_facet => 'Stanford Digital Repository'})
+    it 'has a value for an unrecognized display_type' do
+      hash = described_class.new(access_facet: 'Online',
+                                 druid: druid,
+                                 url_fulltext: "#{purl_url}/#{druid}",
+                                 display_type: 'zzzz',
+                                 building_facet: 'Stanford Digital Repository')
       expect(hash.validate_gdor_fields(mock_config).first).to match(/display_type/)
     end
-    it "should have a value for access_facet other than 'Online'" do
-      hash = GDor::Indexer::SolrDocHash.new({
-        :access_facet => 'BAD',
-        :druid => druid,
-        :url_fulltext => "#{purl_url}/#{druid}",
-        :display_type => 'image', 
-        :building_facet => 'Stanford Digital Repository'})
+    it "has a value for access_facet other than 'Online'" do
+      hash = described_class.new(access_facet: 'BAD',
+                                 druid: druid,
+                                 url_fulltext: "#{purl_url}/#{druid}",
+                                 display_type: 'image',
+                                 building_facet: 'Stanford Digital Repository')
       expect(hash.validate_gdor_fields(mock_config).first).to match(/access_facet/)
     end
-    it "should have a value for building_facet other than 'Stanford Digital Repository'" do
-      hash = GDor::Indexer::SolrDocHash.new({
-        :access_facet => 'Online',
-        :druid => druid,
-        :url_fulltext => "#{purl_url}/#{druid}",
-        :display_type => 'image',
-        :building_facet => 'WRONG'})
+    it "has a value for building_facet other than 'Stanford Digital Repository'" do
+      hash = described_class.new(access_facet: 'Online',
+                                 druid: druid,
+                                 url_fulltext: "#{purl_url}/#{druid}",
+                                 display_type: 'image',
+                                 building_facet: 'WRONG')
       expect(hash.validate_gdor_fields(mock_config).first).to match(/building_facet/)
     end
   end # validate_gdor_fields
 
-  context "#validation_mods" do
+  context '#validation_mods' do
     let(:mock_config) { {} }
-    it 'should have no validation messages for a complete record' do
-      hash = GDor::Indexer::SolrDocHash.new({
-        :modsxml => 'whatever',
-        :title_display => 'title',
-        :pub_year_tisim => 'some year',
-        :author_person_display => 'author',
-        :format_main_ssim => 'Image',
-        :format => 'Image',
-        :language => 'English'
-      })
+    it 'has no validation messages for a complete record' do
+      hash = described_class.new(modsxml: 'whatever',
+                                 title_display: 'title',
+                                 pub_year_tisim: 'some year',
+                                 author_person_display: 'author',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.